docs/source/examples/lre-zne-comparison.md

```diff
@@ -0,0 +1,173 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Comparing LRE and ZNE
+
+Both LRE and ZNE work in two main stages: generate noise-scaled circuits via scaling, and apply inference to resulting measurements post-execution.
+
+This workflow can be executed by a single call to `execute_with_lre` or `execute_with_zne`.
+For resource estimation, Mitiq provides `multivariate_layer_scaling` to inspect the circuits that are to be executed.
+For ZNE we have access to the scaled circuits using the function `scaled_circuits`.
+
+```{danger}
+LRE is currently compatible with quantum programs written using `cirq`.
+Work on making this technique compatible with other frontends is ongoing. 🚧
+```
+
+## Problem Setup
+
+For this demonstration, we'll first define a quantum circuit, and a method of executing circuits for demonstration purposes.
+
+Here we will use the rotated randomized benchmarking circuits on a single qubit and generate 50 random such circuits.
+
+```{code-cell} ipython3
+from mitiq.benchmarks import generate_rotated_rb_circuits
+
+circuits = generate_rotated_rb_circuits(n_qubits=1,
+                                        num_cliffords=3,
+                                        theta=0.7,
+                                        trials=50,
+                                        seed=4)
+
+print(circuits[0])
+```
+
+We define an [executor](../guide/executors.md) which simulates the input circuit subjected to depolarizing noise, and returns the probability of measuring the ground state.
+By altering the value for `noise_level`, ideal and noisy expectation values can be obtained.
+
+```{code-cell} ipython3
+from cirq import DensityMatrixSimulator, depolarize
+
+def execute(circuit, noise_level=0.025):
+    noisy_circuit = circuit.with_noise(depolarize(p=noise_level))
+    rho = DensityMatrixSimulator().simulate(noisy_circuit).final_density_matrix
+    return rho[0, 0].real
+```
+
+```{code-cell} ipython3
+import numpy as np
+
+# Collect ideal and noisy values (probability of measuring 0 across all 100 circuits).
+noisy_values = np.zeros(len(circuits))
+ideal_values = np.zeros(len(circuits))
+
+for i, circuit in enumerate(circuits):
+    noisy = execute(circuit)
+    noisy_values[i] = noisy
+    ideal = execute(circuit, noise_level=0.0)
+    ideal_values[i] = ideal
```

Comment:
correct, I guess it is a habit of mine to always use numpy arrays :). Do you just use sum(list)/len(list) to implement the mean for lists? I want to use mean(list) because that just looks nicer :). I can define a mean function or import from statistics (or some other library).

Reply:
You can still use `np.mean` on the noisy/ideal values, or as you suggested `sum/len` works too!